CHANGELOG.md

```diff
@@ -19,6 +19,13 @@ __________ DO NOT TOUCH ___________
 
 __________ DO NOT TOUCH ___________
 
+<<<<<<< HEAD
+## [x.x.x]
```

Comment:
This needs an update

Reply:
Yes, will update when we're ready to merge.

Reply:
done